src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -428,6 +431,7 @@ export class BasicAutocompleteComponent<O, V = O>
         this.select(undefined);
       }
       this.isInSearchMode.set(false);
+      this.retainSearchValue = "";
```

Comment:
This fixes the issue with search values being kept too long. We did implement something upon an explicit user request to _not_ reset the search during multi-select (so that you can select multiple things under the same search term). But for now, this function here is much better then the confusing previous behavior. So let's go with this :+1: